code/modules/mob/living/carbon/emote.dm

```diff
@@ -127,6 +127,33 @@
 	key_third_person = "winks"
 	message = "winks."
 
+/datum/emote/living/carbon/sweatdrop
+	key = "sweatdrop"
+	key_third_person = "sweatdrops"
+	message = "sweats"
```

Comment:
Make sure to make the message "sweats." with a punctuation at the end. I noticed you missed that from the screenshot.